amd_openvx/openvx/ago/ago_haf_cpu_geometric.cpp

```diff
@@ -736,6 +743,13 @@ int HafCpu_WarpAffine_U8_U8_Nearest_Constant
 				mask.f = _mm_and_ps(mask.f, _mm_cmplt_ps(xmap, srcbx));
 				mask.f = _mm_and_ps(mask.f, _mm_cmpge_ps(ymap, zero));
 				mask.f = _mm_and_ps(mask.f, _mm_cmplt_ps(ymap, srcby));
+				
+				// mask for valid region rectangle
+				mask.f = _mm_and_ps(mask.f, _mm_cmpge_ps(xmap, vl));
+				mask.f = _mm_and_ps(mask.f, _mm_cmplt_ps(xmap, vr));
+				mask.f = _mm_and_ps(mask.f, _mm_cmpge_ps(ymap, vt));
+				mask.f = _mm_and_ps(mask.f, _mm_cmplt_ps(ymap, vb));
+
 				//int m = _mm_movemask_ps(mask.f);
```

Comment:
should we clean-up unused and commented code before the merge? @hansely

Reply:
Those commented code was there even before I started working on. 
I'll talk to Rajy about it and let you know.